contrib/aws/aws-sdk-go/aws/aws.go

```diff
@@ -165,30 +162,38 @@ func awsRegion(req *request.Request) string {
 	return req.ClientInfo.SigningRegion
 }
 
-var serviceTags = map[string]func(params interface{}) (map[string]string, error){
-	"sqs":         sqsTags,
-	"s3":          s3Tags,
-	"sns":         snsTags,
-	"dynamodb":    dynamoDBTags,
-	"kinesis":     kinesisTags,
-	"EventBridge": eventBridgeTags,
-	"states":      sfnTags,
-}
-
-func extraTagsForService(req *request.Request) map[string]string {
+func extraTagsForService(req *request.Request) map[string]interface{} {
```

Comment:
Why did these all go from `map[string]string` to `map[string]interface{}`?

Reply:
The reason is this function is returning a map of tags that apply for specific AWS services. The internal signature for `tracer.Tag` is `(k string, v interface{})`, since tag values could be other than strings in the future.